File: php/0403.php
<?php
# ŠKæ‚ð‹‚ß‚éŠÖ”
function Fact($n)
{
    if ($n <= 1)
        return 1;
    else
        return $n * Fact($n - 1);
}

# 10‚ÌŠKæ
echo Fact(10), "\n";
echo 10 * 9 * 8 * 7 * 6 * 5 * 4 * 3 * 2 * 1, "\n";
?>

File: rb/0403.rb
# ŠKæ‚ğ‹‚ß‚éŠÖ”
def Fact(n)
    if (n <= 1)
        1
    else
        n * Fact(n - 1)
    end
end

# 10‚ÌŠKæ
puts Fact(10)
puts 10 * 9 * 8 * 7 * 6 * 5 * 4 * 3 * 2 * 1
